tensorflow_addons/optimizers/conditional_gradient_test.py

```diff
@@ -14,41 +14,56 @@
 # ==============================================================================
 """Tests for Conditional Gradient."""
 
+import numpy as np
+import pytest
+import platform
+
 import tensorflow as tf
 from tensorflow_addons.utils import test_utils
-import numpy as np
 from tensorflow_addons.optimizers import conditional_gradient as cg_lib
-import platform
 
 
-@test_utils.run_all_in_graph_and_eager_modes
-class ConditionalGradientTest(tf.test.TestCase):
+WINDOWS_CPU = platform.system() == "Windows"
+
+pytestmark = pytest.mark.skipif(
+    WINDOWS_CPU,
```

Comment:
This will skip the whole file on windows. From what I understand, this is not what we want, right? We want to skip only one dtype, which you did in `_dtype_with_checking_system`. Am I missing something?

Reply:
Yes, you are right, thanks for pointing out this, @gabrieldemarmiesse. Updated.